joplin/base/views/new_page_from_modal.py

```diff
@@ -25,98 +26,100 @@ def import_page_from_url(url, jwt_token):
+            raise PermissionDenied
+        # Respond with the id of the new page
+        new_page_id = import_page_from_url(body['title'], body['jwtToken'])
+        response = HttpResponse(json.dumps({'id': new_page_id}), content_type="application/json")
+        return response
+
+    # Pages will go live by default if we don't explicitly specify live=False
+    data = {
+        'live': False
+    }
+    data['title'] = body['title']
+    data['owner'] = request.user
 
-        data = {}
-        data['title'] = body['title']
-        data['owner'] = request.user
+    # Create the page
+    if body['type'] == 'service':
+        page = ServicePage(**data)
+    elif body['type'] == 'information':
+        page = InformationPage(**data)
+    elif body['type'] == 'topic':
+        page = TopicPage(**data)
+    elif body['type'] == 'topiccollection':
+        if body['theme'] is not None:
+            data['theme'] = Theme.objects.get(id=body['theme'])
+        page = TopicCollectionPage(**data)
+    elif body['type'] == 'department':
+        page = DepartmentPage(**data)
+    elif body['type'] == 'documents':
+        page = OfficialDocumentPage(**data)
+    elif body['type'] == 'guide':
+        page = GuidePage(**data)
+    elif body['type'] == 'form':
+        page = FormContainer(**data)
+    elif body['type'] == 'location':
+        page = LocationPage(**data)
+    elif body['type'] == 'event':
+        page = EventPage(**data)
+    elif body['type'] == 'news':
+        page = NewsPage(**data)
 
-        # Create the page
-        if body['type'] == 'service':
-            page = ServicePage(**data)
-        elif body['type'] == 'information':
-            page = InformationPage(**data)
-        elif body['type'] == 'topic':
-            page = TopicPage(**data)
-        elif body['type'] == 'topiccollection':
-            if body['theme'] is not None:
-                data['theme'] = Theme.objects.get(id=body['theme'])
-            page = TopicCollectionPage(**data)
-        elif body['type'] == 'department':
-            page = DepartmentPage(**data)
-        elif body['type'] == 'documents':
-            page = OfficialDocumentPage(**data)
-        elif body['type'] == 'guide':
-            page = GuidePage(**data)
-        elif body['type'] == 'form':
-            page = FormContainer(**data)
-        elif body['type'] == 'location':
-            page = LocationPage(**data)
-        elif body['type'] == 'event':
-            page = EventPage(**data)
-        elif body['type'] == 'news':
-            page = NewsPage(**data)
+    # Add it as a child of home
+    home = HomePage.objects.first()
+    home.add_child(instance=page)
 
-        # Add it as a child of home
-        home = HomePage.objects.first()
-        home.add_child(instance=page)
+    # Save our draft
+    page.save_revision()
 
-        # Save our draft
-        page.save_revision()
-        page.unpublish()  # Not sure why it seems to go live by default
```

Comment:
why dont we need to unpublish anymore?

Reply:
Because now we're setting new pages in create_content_modal to be not live by default.
```
# Pages will go live by default if we don't explicitly specify live=False
    data = {
        'live': False
    }
    data['title'] = body['title']
    data['owner'] = request.user
```